index.ts

```diff
@@ -33,6 +34,7 @@ app.get('/', (req, res) => res.send('Express + TypeScript Server'));
 
 // ACCOUNT
 app.get('/accounts', listAccount);
```

Comment:
Given this answers with multiple accounts, I'd rename `listAccount` to `listAccounts`